### PR TITLE
Implement website layout based on BS-API-DOCS website

### PR DESCRIPTION
**Pull request by hash4di**
Screenshot of current view:
initial_website_layout

### CODE REVIEW

**Comment by wijet** [comment]
:+1:

File: content/index.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<%= render 'head' %>
```
Comment (Fredar) [inline content/index.html]: `Head` and `foot` are already present in layout

File: layouts/foot.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+    </div>
```
Comment (Fredar) [inline layouts/foot.html]: I understand this is present in api-docs, but we can improve that.

First I would move everything(except `head` tags) from head to `default.html`.
Extract `navs` to layouts
Delete `foot` layout
Properly close / set everything in `default.html` - remember to place `yield` correctly.

File: layouts/head.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+          <div class="navbar-header">
+            <button type="button" class="navbar-toggle sidebar-toggle visible-xs-block">
+              <span class="sr-only">Toggle navigation</span>
+              <span class="icon-bar"></span>
+              <span class="icon-bar"></span>
+              <span class="icon-bar"></span>
+            </button>
+            <a class="navbar-brand" href="/">
+              <img alt="BookingSync" src="/images/bookingsync.svg">
+              <span>Developers</span>
```
Comment (ZenCocoon) [inline layouts/head.html]: Best to use `Websites Documentation`

File: layouts/head.html [hunk 4]
```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE HTML>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>BookingSync Websites documentation</title>
```
Comment (ZenCocoon) [inline layouts/head.html]: make `Documentation` with capital as in title

**Comment by Fredar** [comment]
:+1:

**Pull request merged**